llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -966,6 +995,39 @@ Error MCCASBuilder::buildFragments() {
       continue;
     }
 
+    if (Asm.getContext().getObjectFileInfo()->getDwarfLineSection() == &Sec) {
+      SmallVector<char> DebugLineContent;
+      for (const MCFragment &F : Sec) {
+        if (RelocLocation == Atom) {
+          auto Relocs = RelMap.find(&F);
+          if (Relocs != RelMap.end())
+            SectionRelocs.append(Relocs->second.begin(), Relocs->second.end());
+        }
+        ArrayRef<char> FragmentContent;
+        if (F.getKind() == MCFragment::FT_Data)
+          FragmentContent = cast<MCDataFragment>(F).getContents();
+        else if (F.getKind() == MCFragment::FT_Dwarf)
+          FragmentContent = cast<MCDwarfLineAddrFragment>(F).getContents();
+        else
```

Comment:
is the else path an error? What does it mean if we go there?

Reply:
Agree. This should be an assert or unreachable.